pkgs/development/tools/headache/default.nix

```diff
@@ -1,6 +1,9 @@
 { lib, fetchFromGitHub, nix-update-script, ocamlPackages }:
 
-with ocamlPackages;
+let
+  inherit (lib) licenses maintainers;
```

Comment:
Why do you expand the scope of these variables? Declaring variables way before their point of use looks an anti-pattern to me.

Reply:
I've reverted any changes having to do with `lib`.